bin/coach

```diff
@@ -1,57 +1,79 @@
 #!/usr/bin/env ruby
 
-require "slop"
 require "coach/cli/provider_finder"
+require "commander/import"
 
-begin
-  require File.join(Dir.pwd, "config/environment")
-rescue LoadError
-  puts <<~EOS
-    Could not load your Rails app
-    =============================
+program :version, Coach::VERSION
```

Comment:
So does this thing monkeypatch Kernel? Not sure how I feel about that ...

Reply:
ah - good spot, completely missed that! there's an option to use it in a way that it doesn't patch `Kernel`. i'd be keen on trying that option instead of another gem because this one seems to be actively maintained. how do you feel about that?

Reply:
That would be good, yeah. I'm surprised people still do this kind of stuff ...